src/logcollector/read_win_el.c

```diff
@@ -509,6 +510,34 @@ void readel(os_el *el, int printit)
         }
     }
 
+
+    /* Event log was closed and re-opened */
+    else if (id == ERROR_INVALID_HANDLE) {
+        mdebug1("EventLog service has been restarted. Trying to reconnect '%s' channel...", el->name);
```

Comment:
This error is prompted when `EventLog` is restarted and Wazuh tries to read a message from it. I'm not aware whether it could appear if something else happens (we haven't seen that behavior before). Anyway, the message could be changed just in case.